static/styles/settings.css

```diff
@@ -702,8 +702,6 @@ input[type="checkbox"] {
 #account-settings-status {
     text-align: center;
     width: 50%;
```

Comment:
This is also redundant. since width property for `alert-notification` has `!important` tag.

Reply:
And I think `text-align` also has no effect since width is always set as auto covering the content. Cannot see these properties for indicator used in other pages as well.